manim/config.py

```diff
@@ -37,22 +38,22 @@ def parse_cli():
         parser.add_argument(
             "-l", "--low_quality",
             action="store_true",
-            help="Render at a low quality (for faster rendering)",
+            help="Render at low quality (for fastest rendering)",
```

Comment:
The comments on the render speed aren't really needed; if anything we should remove the one that's there.

Reply:
That's what I would've thought initially, but then there's the problem of "high quality".
Users will by default not assume that "high quality" is under default "production quality". That's why I added the explanations. If we take them off we should compensate in some other way I can't think of right now.

Reply:
You mean as in... getting rid of the -l -m flags and so forth?

Reply:
Please refer to and participate in #65

Reply:
I replied there, so I'll resolve it here.